README.md

```diff
@@ -52,25 +62,29 @@ cd one-app
 npm ci --no-optional
 ```
 
-**Build a Module:**
+**Serve your Module to One App**
+
+At the root of your `one-app` repo, run:
 
 ```bash
-cd prod-sample/sample-modules/frank-lloyd-root/0.0.0
-npm ci
-cd ../../../..
-npm run serve-module prod-sample/sample-modules/frank-lloyd-root/0.0.0
-# this is used by the frank-lloyd-root module
-export ONE_CONFIG_ENV=development
+npm run serve-module <local-path-to-generated-module>
+# e.g. npm run serve-module ../my-first-module
 ```
 
-**Declare a Root Module and Start One App:**
+The `serve-module` command generates a `static` folder in the `one-app` root directory, containing a `module-map.json` and a `modules` folder with your bundled module code. Paired with the built-in [one-app-dev-cdn](https://github.com/americanexpress/one-app-dev-cdn) library, you're able to utilize [holocron]() while running your entire One App instance locally. No need to deploy and fetch remote assets from a CDN at this step.
+
+**Declare the module as your Root Module and start One App:**
+
+Start up One App and declare your new module as the [Root Module](#-the-root-module):
 
 ```bash
-npm start -- --root-module-name=frank-lloyd-root
+npm start -- --root-module-name=<module-name>
+# e.g. npm start -- --root-module-name=my-first-module
 ```
 
-This starts One App and makes it available at http://localhost:3000/success where you can see it in action!
+This starts One App and makes it available at http://localhost:3000/success where you can see it in action! 
```

Comment:
This line can be updated since the generated module only has an index route right? So `https://localhost:3000/success` should be `https://localhost:3000/`.

Reply:
fixed 030f665